extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php

```diff
@@ -211,18 +211,25 @@ protected function getImageFromPageId( $mPageId ) {
 	}
 
 	protected function getTemplateVarsForItem( $pageId ) {
+		global $wgScriptPath;
+
 		$oTitle = Title::newFromID( $pageId );
 		if ( !( $oTitle instanceof Title ) ) {
 			return false;
 		}
 
 		$oMemCache = F::App()->wg->memc;
-		$sKey = wfMemcKey(
+		$keyParams = [
 			'category_exhibition_category_cache_1',
 			$pageId,
 			$this->cacheHelper->getTouched( $oTitle )
-		);
+		];
 
+		if ( !empty( $wgScriptPath ) ) {
+			array_push( $keyParams, $wgScriptPath );
```

Comment:
For adding one variable to an array `$keyParams[] = $wgScriptPath;` is preferred (including in PHP docs).

Reply:
I had a brain fart and forgot about this syntax :man_facepalming: Thanks, updated.